### PR TITLE
Demo Package

### PR DESCRIPTION
**Pull request by DavidMansolino**
The Goal of this PR is to add a `demos` ROS2 package that will showcase all the other packages by providing a nice simulation and include control (which is not included in the other packages).

This is the last remaining point before being able to make a public release of webots_ros2 1.0.0 :tada:

### CODE REVIEW

**Comment by DavidMansolino** [comment]
UR Control command to add to control node: 
```
ros2 action send_goal /ur/follow_joint_trajectory control_msgs/action/FollowJointTrajectory "{
  trajectory: {
    joint_names: [shoulder_pan_joint, shoulder_lift_joint, elbow_joint, wrist_1_joint, finger_1_joint_1, finger_2_joint_1, finger_middle_joint_1],
    points: [
      { 
        positions: [0.0, 0.0, 0.0, 0.0, 0.85, 0.85, 0.6],
        velocities: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1],
        accelerations: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1],
        time_from_start: { sec: 10, nanosec: 0 }
      },
      { 
        positions: [0.63, -2.26, -1.88, -2.14, 0.85, 0.85, 0.6],
        velocities: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1],
        accelerations: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1],
        time_from_start: { sec: 20, nanosec: 0 }
      },
      { 
        positions: [0.63, -2.26, -1.88, -2.14, 0.0, 0.0, 0.0],
        velocities: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1],
        accelerations: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1],
        time_from_start: { sec: 30, nanosec: 0 }
      },
      { 
        positions: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
        velocities: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1],
        accelerations: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1],
        time_from_start: { sec: 40, nanosec: 0 }
      },
    ]
  }
}"
```

**Comment by DavidMansolino** [comment]
:+1:

**Comment by fabienrohrer** [comment]
👀

File: webots_ros2_demos/worlds/armed_robots.wbt [hunk 1]
```diff
@@ -0,0 +1,563 @@
...
+          url [
+            "textures/corrugated_plates_occlusion.jpg"
+          ]
+        }
+        textureTransform TextureTransform {
+          scale 4 3
+        }
+      }
+      geometry Plane {
+        size 10.1 7.6
```
Comment (fabienrohrer) [inline webots_ros2_demos/worlds/armed_robots.wbt]: This doesn't match with the floor:

```suggestion
        size 8.2 5.8
```

File: webots_ros2_demos/worlds/armed_robots.wbt [hunk 2]
```diff
@@ -0,0 +1,563 @@
...
+        }
+      }
+      geometry Plane {
+        size 5.8 3
+      }
+    }
+  ]
+}
+DEF ROOF Transform {
+  translation 0 3 0
```
Comment (fabienrohrer) [inline webots_ros2_demos/worlds/armed_robots.wbt]: This doesn't match with the floor:

```suggestion
  translation 0.79 3 -0.69
```

File: webots_ros2_demos/worlds/armed_robots.wbt [hunk 3]
```diff
@@ -0,0 +1,563 @@
...
+DEF POLE Solid {
+  translation -2.47 2.9 -3.59
+  rotation 1 0 0 1.5708
+  children [
+    USE ROOF_POLE_SHAPE
+  ]
+  name "solid(9)"
+}
+WoodenPalletStack {
+  translation 4.27993 -1.29674e-13 -2.83696
```
Comment (fabienrohrer) [inline webots_ros2_demos/worlds/armed_robots.wbt]: Several fields could be rounded in the world file.

**Pull request merged**